syntax = "proto3";
package proto;

message Help {
  //引数の説明
  string usage = 1;
  //短い説明
  string short_description = 2;
  //長い説明
  string long_description = 3;
  //ソースのurl
  string source_url = 4;
}
